.travis.yml

```diff
@@ -2,11 +2,10 @@ sudo: false
 
 language: scala
 scala:
-  - 2.11.12
-  - 2.12.8
-  - 2.13.0
+  - 2.12.10
+  - 2.13.1
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
Can we choose `openjdk8` instead? I'd prefer if we kept sure it works on JVM 8.

Reply:
The first build was run with `oraclejdk8` (I didn't change anything in this regard), and it failed:
https://travis-ci.org/ruippeixotog/akka-testkit-specs2/jobs/624624787?utm_medium=notification&utm_source=github_status

Following this error: `Expected feature release number in range of 9 to 14, but got: 8`, I tried with `oraclejdk9` and it succeeded. 

I'll try with `openjdk8` instead as you suggested.

Reply:
Passed: https://travis-ci.org/ruippeixotog/akka-testkit-specs2/builds/624681121?utm_medium=notification&utm_source=github_status